### ISSUE TITLE
Plasma 6 Support

### ISSUE THREAD

**Issue by c3p0vsr2d2** [qewer33/Exquisite#36]
Does this work on Plasma 6? Should I hold off on updating Plasma?

**Comment by artkoz** [comment]
Doesn't work, unfortunately.

**Comment by zelch** [comment]
I strongly suspect that it will be ported as soon as either qwer33's distribution gets Plasma 6, or mine does, whichever comes first.

I'm sure that if anyone else is both interested and able, patches would be welcome.

(I'm on Debian SID, and I'm not aware of any public timelines.  Obviously, well before any stable Debian releases with it, but.)

**Comment by qewer33** [comment]
I'm closing this issue in favor of https://github.com/qewer33/ClearClock/issues/15, please check my comments there

**Comment by qewer33** [comment]
Oops, I just realized I mixed issues from two different projects. Re-opening this again, sorry!

**Comment by imthenachoman** [comment]
This makes me wonder what distro @qewer33 and @zelch use?

**Comment by zelch** [comment]
@imthenachoman I'm on Debian Sid, AKA unstable, the constantly churning development 'release'.

I keep seeing more Plasma 6 components showing up, so I'm hopeful that plasma-desktop will make an appearance in the next month.

**Comment by imthenachoman** [comment]
@zelch Ah. Nice. Is it on Plasma 5 or 6? You don't notice stability issues with it?

I moved from Debian stable to Kubuntu cause I wanted something more updated. I just installed openSUSE Tumbleweed on a second partition but it comes with Plasma 6. Since Exquisite isn't available for it yet, I'll stick with 5. But I'm in no hurry. I didn't see anything in Plasma 6 that made it desirable to switch.

**Comment by zelch** [comment]
It's currently still on Plasma 5.

As far as stable?  Well, more or less by definition, no, but as some background...

I've been running Debian unstable as my daily driver on my personal machine for something close to 20 years, and I'd have to check some stuff to see which side of it I'm on at this point.

Once upon a time, when there was far more free time and energy, I was a Debian Developer.

I _don't_ recommend that people run Debian unstable unless they feel perfectly comfortable untangling complicated dependency problems in aptitude (because large migrations happen, and things can get really messy in the middle of them), and are at least vaguely confident in rescuing their system if things really go sideways.

I haven't had the latter happen in a _very_ long time, but part of that is that I'm quite comfortable with what I'm doing.

**Comment by imthenachoman** [comment]
I'm talking to a Debian developer! How cool! 

Thanks for the perspective. I'll stick with Kubuntu for now but may switch to OpenSUSE TW when Exquisite is ported. But I'm in no rush.

**Comment by Jafner** [comment]
Not sure if this should be in a separate issue, but it would be helpful to have Plasma 5 dependence documented in the README.  

I'm new to KDE and hopped straight into Plasma 6 without realizing how wide a gap there was from 5 re: ecosystem support. It's been confusing and frustrating sometimes to encounter cool projects that are silently dependent on Plasma 5.  

Regardless, thanks for your work!

**Comment by imthenachoman** [comment]
@Jafner I agree. I switched to OS TW with KDE 6 but then I realized Exquisite isn't ready for KDE 6 so I went back to Debian stable with KDE 5. KDE 6 doesn't have anything ground breaking that makes it worth not having Exquisite for me.

**Comment by imthenachoman** [comment]
Is there an update to if this works on KDE 6 yet?

**Comment by qewer33** [comment]
> Is there an update to if this works on KDE 6 yet?

Sadly no :(
Last time I tried the Kwin 6 API was incredibly buggy and the new breaking API changes were nowhere near properly documented. I might try porting it again later this month.
In the meantime, I think kzones works on Plasma 6 and is similar to Exquisite: https://github.com/gerritdevriese/kzones

**Comment by imthenachoman** [comment]
Thanks. I will check it out. But I do love Exquisite. It is, you could say, [exquisite](https://www.merriam-webster.com/dictionary/exquisite).
